assets/css/_route_ladder.scss

```diff
@@ -49,7 +49,7 @@
 
   .card-body {
     display: inline-grid;
-    grid-template-columns: 1fr 1fr auto 1fr 1fr;
+    grid-template-columns: 1fr auto auto auto 1fr;
```

Comment:
<img width="313" alt="Screenshot 2024-05-29 at 2 38 32 PM" src="https://github.com/mbta/skate/assets/912020/36513c9b-fcca-4b26-ad89-075233e7a8b7">

It looks like the route pill got knocked off-center?

Reply:
My guess is that the change from `1fr` to `auto` isn't necessary, and that the change to `margin` is all that's needed, but I'm not going to pretend to know enough about `grid` to be certain of that.

Reply:
Sadly, ended up needing a bigger change. Shifting to `auto` was needed to keep the centering nice with the dropdown. Since that was precarious, had to rework a bit